scripts/generate.mjs

```diff
@@ -0,0 +1,78 @@
+import { Octokit } from 'octokit';
+import * as fs from 'node:fs/promises';
+
+const gh = new Octokit({
+    auth: process.env.GITHUB_TOKEN || process.env.GH_TOKEN
+});
+
+const searchResults = gh.paginate.iterator(gh.rest.search.repos, {
+    q: 'titanium in:topics language:objc language:swift language:java language:kotlin language:javascript',
+    per_page: 100
+});
+
+const repositories = [];
+for await (const {data} of searchResults) {
+    const mappedData = await Promise.all(data.map(mapData));
```

Comment:
This is going to be a problem. You have 100 records that you are simultaneously requesting making 3 getContent calls. As much as it pains us, we should iterate over `data` serially with a `setTimeout(..., 2000)` just to make sure we don't hit the rate limiting. Remember, the `GITHUB_TOKEN` is shared with the downloads site which runs whenever we do an SDK release or a CI build.

Reply:
I think we might get away with not worrying about that as octokit includes the retry and throttling plugins by default. If we hit issues we could just bump the retry count in the config maybe

Reply:
I dunno, I ran into serious rate limiting when regenerating the downloads site.

Reply:
That uses octokit from `@actions/core` right? That looks to not include the retry/throttle plugins https://github.com/actions/toolkit/blob/main/packages/github/package.json#L41-L44

`octokit` wraps the dependency that is used by the `@actions/core` package up and adds that throttling and stuff for us https://github.com/octokit/octokit.js